At least three files have Python syntax errors

$ `flake8 . --count --select=E9,F63,F7 --show-source --statistics`
```
./molequeue/python/test/clienttest.py:24:11: E999 SyntaxError: invalid syntax
    print "MoleQueue ID: ", molequeue_id
          ^
./molequeue/molequeue/app/testing/scripts/submitJob.py:19:11: E999 SyntaxError: invalid syntax
    print "Client %s connecting to socket: %s"%(clientId, socketname)
          ^
./molequeue/molequeue/app/testing/scripts/sendRpcKill.py:13:11: E999 SyntaxError: invalid syntax
    print "Connecting to socket: %s"%socketname
          ^
3     E999 SyntaxError: invalid syntax
3
```

--- a/molequeue/app/testing/scripts/sendRpcKill.py
+++ b/molequeue/app/testing/scripts/sendRpcKill.py
@@ -10,7 +10,7 @@ socketname = ""
 def run_test():
   global socketname
   if debug:
-    print "Connecting to socket: %s"%socketname
+    print ("Connecting to socket: %s"%socketname)
   client = mq.Client()
   client.connect_to_server(socketname)
   res = client._send_rpc_kill_request(5)
@@ -32,10 +32,10 @@ def main(argv):
   try:
     test_result = run_test()
   except Exception as ex:
-    print ex
+    print (ex)
     test_result = 1
   if debug:
-    print "Exiting with code: %d"%test_result
+    print ("Exiting with code: %d"%test_result)
   sys.stdout.flush()
   return test_result
 

--- a/molequeue/app/testing/scripts/sendRpcKill.py
+++ b/molequeue/app/testing/scripts/sendRpcKill.py
@@ -10,7 +10,7 @@ socketname = ""
 def run_test():
   global socketname
   if debug:
-    print ("Connecting to socket: %s"%socketname)
+    print("Connecting to socket: %s"%socketname)
   client = mq.Client()
   client.connect_to_server(socketname)
   res = client._send_rpc_kill_request(5)
@@ -32,10 +32,10 @@ def main(argv):
   try:
     test_result = run_test()
   except Exception as ex:
-    print (ex)
+    print(ex)
     test_result = 1
   if debug:
-    print ("Exiting with code: %d"%test_result)
+    print("Exiting with code: %d"%test_result)
   sys.stdout.flush()
   return test_result
 

--- a/molequeue/app/testing/scripts/submitJob.py
+++ b/molequeue/app/testing/scripts/submitJob.py
@@ -16,7 +16,7 @@ mq_id_lock = Lock()
 def run_test():
   global socketname, molequeue_ids_done, clientId, mq_id_lock, num_jobs
   if debug:
-    print "Client %s connecting to socket: %s"%(clientId, socketname)
+    print ("Client %s connecting to socket: %s"%(clientId, socketname))
   client = mq.Client()
   client.connect_to_server(socketname)
 
@@ -29,9 +29,9 @@ def run_test():
           with mq_id_lock:
             molequeue_ids_done.append(moleQueueId)
           if debug:
-            print "Job %d finished! (Client %s)"%(moleQueueId, clientId)
+            print ("Job %d finished! (Client %s)"%(moleQueueId, clientId))
     except Exception as ex:
-      print "Unexpected notification:", msg, ex
+      print ("Unexpected notification:", msg, ex)
     sys.stdout.flush()
 
   client.register_notification_callback(notification_callback)
@@ -50,15 +50,15 @@ def run_test():
       client.disconnect()
       raise Exception("Connection timed out!")
     if debug:
-      print "Submitted job %d (Client %s)"%(molequeue_id, clientId)
+      print ("Submitted job %d (Client %s)"%(molequeue_id, clientId))
     sys.stdout.flush()
 
   timeout = 30
   mq_id_lock.acquire()
   while len(molequeue_ids) != len(molequeue_ids_done) and timeout > 0:
     if debug:
-      print "Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
-        (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done))
+      print ("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
+        (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done)))
     sys.stdout.flush()
     timeout -= 1
     mq_id_lock.release()
@@ -90,10 +90,10 @@ def main(argv):
   try:
     test_result = run_test()
   except Exception as ex:
-    print ex
+    print (ex)
     test_result = 1
   if debug:
-    print "Exiting with code: %d (client %s)"%(test_result, clientId)
+    print ("Exiting with code: %d (client %s)"%(test_result, clientId))
   sys.stdout.flush()
   return test_result
 

--- a/molequeue/app/testing/scripts/submitJob.py
+++ b/molequeue/app/testing/scripts/submitJob.py
@@ -16,7 +16,7 @@ mq_id_lock = Lock()
 def run_test():
   global socketname, molequeue_ids_done, clientId, mq_id_lock, num_jobs
   if debug:
-    print ("Client %s connecting to socket: %s"%(clientId, socketname))
+    print("Client %s connecting to socket: %s"%(clientId, socketname))
   client = mq.Client()
   client.connect_to_server(socketname)
 
@@ -29,9 +29,9 @@ def run_test():
           with mq_id_lock:
             molequeue_ids_done.append(moleQueueId)
           if debug:
-            print ("Job %d finished! (Client %s)"%(moleQueueId, clientId))
+            print("Job %d finished! (Client %s)"%(moleQueueId, clientId))
     except Exception as ex:
-      print ("Unexpected notification:", msg, ex)
+      print("Unexpected notification:", msg, ex)
     sys.stdout.flush()
 
   client.register_notification_callback(notification_callback)
@@ -50,14 +50,14 @@ def run_test():
       client.disconnect()
       raise Exception("Connection timed out!")
     if debug:
-      print ("Submitted job %d (Client %s)"%(molequeue_id, clientId))
+      print("Submitted job %d (Client %s)"%(molequeue_id, clientId))
     sys.stdout.flush()
 
   timeout = 30
   mq_id_lock.acquire()
   while len(molequeue_ids) != len(molequeue_ids_done) and timeout > 0:
     if debug:
-      print ("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
+      print("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
         (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done)))
     sys.stdout.flush()
     timeout -= 1
@@ -90,10 +90,10 @@ def main(argv):
   try:
     test_result = run_test()
   except Exception as ex:
-    print (ex)
+    print(ex)
     test_result = 1
   if debug:
-    print ("Exiting with code: %d (client %s)"%(test_result, clientId))
+    print("Exiting with code: %d (client %s)"%(test_result, clientId))
   sys.stdout.flush()
   return test_result
 

--- a/molequeue/app/testing/scripts/submitJob.py
+++ b/molequeue/app/testing/scripts/submitJob.py
@@ -57,7 +57,7 @@ def run_test():
   mq_id_lock.acquire()
   while len(molequeue_ids) != len(molequeue_ids_done) and timeout > 0:
     if debug:
-      print("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%\
+      print("Client %s waiting to finish (timeout=%d unmatchedIDs=%d)"%
         (clientId, timeout, len(molequeue_ids) - len(molequeue_ids_done)))
     sys.stdout.flush()
     timeout -= 1

--- a/python/test/clienttest.py
+++ b/python/test/clienttest.py
@@ -21,7 +21,7 @@ class TestClient(unittest.TestCase):
 
     molequeue_id = client.submit_job(job)
 
-    print ("MoleQueue ID: ", molequeue_id)
+    print("MoleQueue ID: ", molequeue_id)
 
     self.assertTrue(isinstance(molequeue_id, int))
 
@@ -105,7 +105,7 @@ class TestClient(unittest.TestCase):
     queues = client.request_queue_list_update()
 
     for q in queues:
-      print (q.name, ", ", q.programs)
+      print(q.name, ", ", q.programs)
 
     client.disconnect()
 

--- a/python/test/clienttest.py
+++ b/python/test/clienttest.py
@@ -21,7 +21,7 @@ class TestClient(unittest.TestCase):
 
     molequeue_id = client.submit_job(job)
 
-    print "MoleQueue ID: ", molequeue_id
+    print ("MoleQueue ID: ", molequeue_id)
 
     self.assertTrue(isinstance(molequeue_id, int))
 
@@ -105,7 +105,7 @@ class TestClient(unittest.TestCase):
     queues = client.request_queue_list_update()
 
     for q in queues:
-      print q.name, ", ", q.programs;
+      print (q.name, ", ", q.programs)
 
     client.disconnect()
 

